src/libaktualizr-c/test/api-test.c

```diff
@@ -27,7 +27,7 @@ int main(int argc, char **argv) {
 
   c = Aktualizr_campaigns_check(a);
   if (c) {
-    printf("Accepting running campaign\n");
```

Comment:
Why the change from `printf` to `fprintf(stdout, ...)`?

Reply:
Reverted.